modular.d.ts

```diff
@@ -131,6 +132,25 @@ export declare const MsgPack: unknown;
  */
 export declare const RealtimePresence: unknown;
 
+/**
+ * The module is experimental and the API is subject to change.
```

Comment:
we reverted the "modules" mechanism for modular variant of the library to use "plugins" again as it was in v1.
see: https://github.com/ably/ably-js/pull/1663
mainly it means that instead of "module" we call them "plugin", and to use a plugin in your modular variant you do:
```typescript
const realtime = new BaseRealtime({ ...options, plugins: { ChannelGroups } });
```
instead of
```typescript
const realtime = new BaseRealtime(options, { ChannelGroups });
```

you can take a look at the linked PR and jsdocs for other plugins in this file to find the documentation differences.
Basically, we just need to change documentation here.

Reply:
Yeah I updated the implementation, just need to update these docs :)